### PR TITLE
New enhancements on organization page

### PR DESCRIPTION
**Pull request by MhdBarazi**

### CODE REVIEW

**Comment by alayham** [comment]
This update broke a test
OrganizationPageComponent Display the name of the organization

if you do not want to display the org name as a field, you can remove the test. I recommend keeping the org name as a field, because the title can be changed if required

**Comment by MhdBarazi** [comment]
@alayham Try now my branch. New update!

File: src/app/admin/organization-page/organization-page.component.spec.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ describe('OrganizationPageComponent', () => {
   it('Display the name of the organization',() => {
     component.organizationId = testData.orgList[index].id;
     fixture.detectChanges();
-    de = fixture.debugElement.query(By.css('div.org-name'));
+    de = fixture.debugElement.query(By.css('h2.org-title'));
```
Comment (alayham) [inline src/app/admin/organization-page/organization-page.component.spec.ts]: this change makes this test identical to the previous test. you should either remove this test, or display the organization name as a field.

**Comment by alayham** [comment]
Why are we changing interfaces in this branch?

**Comment by albannurkollari** [comment]
@alayham It's a rebase thing. My commit is included in his branch/pull request. Nothing to worry cause it wont overwrite nothing else but his own branches changed files.

**Comment by alayham** [comment]
@albannurkollari why are we changing interfaces at all?

I proposed a solution to close projects without any change in interfaces. did anybody check it?

**Comment by albannurkollari** [comment]
@alayham I checked the proposed solution. I don't like it (personally). But we all others came to a common sense that we need more properties in the interfaces. So to speak the interfaces I created aren't used anywhere, its for testing purpose also. And why have it on development? Because many other can be able to use them if need be. Btw this is my last comment in here as its becoming very unusual to discuss such matters in here. Either Approve/Not this branch. We all seen @MhdBarazi work, and it's not breaking any test nor runtime errors.

**Comment by alayham** [comment]
If something does not break, it does not mean it is good. anyway, it is your code. Do what you like. 

As for me, I prefer to avoid unnecessary maintenance and redundant information in DB.

**Comment by MhdBarazi** [comment]
I will not merge any new changes until i get the approval from all Team, so it will be in pending status until we agree all on this.

**Pull request closed without merging**